delphi/GrFN/networks.py

```diff
@@ -171,15 +157,19 @@ class GroundedFunctionNetwork(ComputationalGraph):
     node.
     """
 
-    def __init__(self, G, scope_tree):
+    def __init__(self, G, scope_tree, outputs):
         super().__init__(G)
+        self.outputs = outputs
         self.scope_tree = scope_tree
+        for name, data in self.nodes(data=True):
+            if len(data) == 0:
+                print(name, data)
         self.inputs = [
             n
             for n, d in self.in_degree()
             if d == 0 and self.nodes[n]["type"] == "variable"
         ]
-        self.output_node = self.outputs[-1]
+        # self.output_node = self.outputs[-1]
```

Comment:
Can we get rid of this commented-out line?

Reply:
Done!